usaspending_api/data/ContractSummary_download_readme.txt

```diff
@@ -0,0 +1,31 @@
+========ABOUT THESE FILES========
+
+This ZIP file was generated from a specific Contract Award Summary Page on USAspending.gov, located at [CAN WE FILL THIS IN???]

```

Comment:
Can we replace or remove `[CAN WE FILL THIS IN???]` before merging this over?

Reply:
yeah that's fair lol